presto-docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -221,6 +221,7 @@ VARINT            VARCHAR
 SMALLINT          INTEGER
 TINYINT           INTEGER
 DATE              DATE
+DURATION          VARBINARY
```

Comment:
In the definition `DURATION(createUnboundedVarcharType(), null)`, `createUnboundedVarcharType()` returns `VARCHAR`. The type should be `VARCHAR` instead of `VARBINARY`?
Could you also add a test to show others how DURATION is used? Thanks!